notebooks/00_quick_start/sar_movielens_with_azureml.ipynb

```diff
@@ -0,0 +1,615 @@
+{
```

Comment:
<p>is it possible to who the output of the the metrics? you are logging them in the script with azureml</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/575/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
Yes. It's right below one of the screenshot. I retook the screenshot to include the metrics

Reply:
hey @jingyanwangms I can't see the new screenshot, are you sure is there?